core/trino-main/src/test/java/io/trino/sql/analyzer/TestScope.java

```diff
@@ -16,14 +16,14 @@
 import io.trino.sql.tree.DereferenceExpression;
 import io.trino.sql.tree.Expression;
 import io.trino.sql.tree.QualifiedName;
-import org.testng.annotations.Test;
+import org.junit.jupiter.api.Test;
 
 import java.util.Optional;
 
 import static io.trino.spi.type.BigintType.BIGINT;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertFalse;
-import static org.testng.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
The argument order of `assertEquals` is different between TestNG and JUnit. Please see #9362 for the reference.

Reply:
@ebyhr  I see, so in that case I should replace the order of the arguments in all the **assertEquals** statements ? for eg:
 
 Change: 

`assertEquals(outer.tryResolveField(c1).get().getField(), outerColumn1);`

To:

`assertEquals(outerColumn1, outer.tryResolveField(c1).get().getField() );`

?

Reply:
Yes, we should replace all of them.

Reply:
Got it, and thanks for all the help. Appreciate it!